core/domain/user_services.py

```diff
@@ -513,7 +873,17 @@ def update_email_preferences(
 
     If no UserEmailPreferencesModel exists for this user, a new one will
     be created.
+
+    Args:
+        user_id: str. The user id.
+        can_receive_email_updates: bool. Indicates whether or not User should
```

Comment:
Here and below: strike "Indicates." We don't use that on any other arg docstring, so you can just say "Whether the given user can receive email updates.", etc.